tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+skipsdist=True
+envlist = clean,ansible{2.12,2.13}-py{38,39,310}-{with_constraints,without_constraints},linters
```

Comment:
I believe we can focus on 3.9+ without taking too much risk. You can drop the `-{with_constraints,without_constraints}` part. We don't need that for this one.
Also, I'm not so sure about `ansible{2.12,2.13}`, we won't use them for now.

basically, I would just use `envlist = py{39,310,311}`